MOTIVATION.md

```diff
@@ -0,0 +1,138 @@
+# Motivation
+
+The Human Cell Atlas (HCA) Data Coordination Platform (DCP) ingests,
+processes, and stores single-cell data with the goal of making that data easily
+available to a wide audience of researchers.
+The single-cell data type of greatest interest to most researchers is expression
+matrices, matrices that contain cell-by-gene expression values that are the
+starting point for many downstream analyses.
+The importance of this data type calls for a specialized interface that
+maximizes its utility. This specialized interface is called the DCP "matrix
+service". The matrix service is responsible for enabling investigators to access
+and analyze HCA expression data as easily as possible.
+
+# Expression Matrix Interfaces
+
+The native interface exposed by the DCP storage system, along with the quirks of
+the DCP's data organization, would make working with the matrices challenging.
+In contrast, the matrix service aims to incorporate HCA expression data into
```

Comment:
_aims to_ sounds aspirational.

``the matrix service allows researchers to seamlessly incorporate HCA expression data into their existing analysis practices.``

I am fond of split infinitives.

Reply:
:+1: